# Translations template for vigilo-connector.
# Copyright (C) 2012 ORGANIZATION
# This file is distributed under the same license as the vigilo-connector
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2012.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: vigilo-connector 2.1.2\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2012-09-17 11:39+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/connector/amqp.py:88
#, python-format
msgid "Channel open failed: %s"
msgstr ""

#: src/vigilo/connector/amqp.py:92
#, python-format
msgid "Error getting channel: %s"
msgstr ""

#: src/vigilo/connector/amqp.py:96
#, python-format
msgid "AMQP authentication failed: %s"
msgstr ""

#: src/vigilo/connector/amqp.py:100
#, python-format
msgid "Channel %(id)d was unexpectedly closed, disconnecting. Reason: %(reason)s"
msgstr ""

#: src/vigilo/connector/client.py:85
#, python-format
msgid "Server %(oldserver)s did not answer after %(attempts)d attempts"
msgstr ""

#: src/vigilo/connector/client.py:280
#, python-format
msgid "Lost connection to the bus (%s)"
msgstr ""

#: src/vigilo/connector/client.py:362
#, python-format
msgid "Sending failed: %(reason)s"
msgstr ""

#: src/vigilo/connector/client.py:383 src/vigilo/connector/client.py:589
#, python-format
msgid "Invalid configuration value for option \"%(key)s\"."
msgstr ""

#: src/vigilo/connector/client.py:388 src/vigilo/connector/client.py:594
#, python-format
msgid "Missing configuration value for option \"%(key)s\"."
msgstr ""

#: src/vigilo/connector/client.py:463
#, python-format
msgid "Creating lock file in %s"
msgstr ""

#: src/vigilo/connector/client.py:466
#, python-format
msgid "Lock file successfully created in %s"
msgstr ""

#: src/vigilo/connector/client.py:471
msgid "Error: lockfile found, another instance may be running."
msgstr ""

#: src/vigilo/connector/client.py:484
#, python-format
msgid "Error %(code)s: %(message)s"
msgstr ""

#: src/vigilo/connector/client.py:491
#, python-format
msgid "Error: %s"
msgstr ""

#: src/vigilo/connector/client.py:497
msgid "Timeout"
msgstr ""

#: src/vigilo/connector/client.py:556
msgid "No handler registered for this one-shot client"
msgstr ""

#: src/vigilo/connector/conffile.py:76
msgid "Received signal to reload the configuration"
msgstr ""

#: src/vigilo/connector/conffile.py:89
msgid "No configuration yet!"
msgstr ""

#: src/vigilo/connector/configure.py:57
#, python-format
msgid "Creating queue %s"
msgstr ""

#: src/vigilo/connector/configure.py:63
#, python-format
msgid "Deleting queue %s"
msgstr ""

#: src/vigilo/connector/configure.py:68
#, python-format
msgid "Creating exchange %(exchange)s of type %(type)s"
msgstr ""

#: src/vigilo/connector/configure.py:75
#, python-format
msgid "Deleting exchange %s"
msgstr ""

#: src/vigilo/connector/configure.py:83
#, python-format
msgid "Subscribing queue %(queue)s to exchange %(exchange)s (key: %(key)s)"
msgstr ""

#: src/vigilo/connector/configure.py:94
#, python-format
msgid "Unsubscribing queue %(queue)s from exchange %(exchange)s (key: %(key)s)"
msgstr ""

#: src/vigilo/connector/configure.py:106 src/vigilo/connector/configure.py:107
#, python-format
msgid "Can't find file '%s'"
msgstr ""

#: src/vigilo/connector/configure.py:116
#, python-format
msgid "Exchange %(name)s (%(type)s)"
msgstr ""

#: src/vigilo/connector/configure.py:127
#, python-format
msgid "Queue %s"
msgstr ""

#: src/vigilo/connector/configure.py:130
#, python-format
msgid "Queue %(queue)s subscribed to exchange %(exchange)s (key: %(key)s)"
msgstr ""

#: src/vigilo/connector/configure.py:145
msgid "Server address"
msgstr ""

#: src/vigilo/connector/configure.py:147
msgid "Bus user"
msgstr ""

#: src/vigilo/connector/configure.py:149
msgid "Bus password"
msgstr ""

#: src/vigilo/connector/configure.py:151
msgid "Show debugging information."
msgstr ""

#: src/vigilo/connector/configure.py:156
msgid "Vigilo bus configurator"
msgstr ""

#: src/vigilo/connector/configure.py:157
msgid "Subcommands"
msgstr ""

#: src/vigilo/connector/configure.py:163
msgid "Create an AMQP queue on the server."
msgstr ""

#: src/vigilo/connector/configure.py:165 src/vigilo/connector/configure.py:173
#: src/vigilo/connector/configure.py:199 src/vigilo/connector/configure.py:209
msgid "Queue name"
msgstr ""

#: src/vigilo/connector/configure.py:171
msgid "Delete an AMQP queue on the server."
msgstr ""

#: src/vigilo/connector/configure.py:179
msgid "Create an AMQP exchange on the server."
msgstr ""

#: src/vigilo/connector/configure.py:182
msgid "Exchange type"
msgstr ""

#: src/vigilo/connector/configure.py:183 src/vigilo/connector/configure.py:191
#: src/vigilo/connector/configure.py:200 src/vigilo/connector/configure.py:210
msgid "Exchange name"
msgstr ""

#: src/vigilo/connector/configure.py:189
msgid "Delete an AMQP exchange on the server."
msgstr ""

#: src/vigilo/connector/configure.py:197
msgid "Subscribe a queue to an exchange."
msgstr ""

#: src/vigilo/connector/configure.py:201 src/vigilo/connector/configure.py:211
msgid "Routing key"
msgstr ""

#: src/vigilo/connector/configure.py:207
msgid "Unsubscribe a queue from an exchange."
msgstr ""

#: src/vigilo/connector/configure.py:217
msgid "Reads the configuration from an INI file."
msgstr ""

#: src/vigilo/connector/configure.py:219
msgid "Configuration file"
msgstr ""

#: src/vigilo/connector/handlers.py:116
#, python-format
msgid "Could not initialize the queue: %(reason)s"
msgstr ""

#: src/vigilo/connector/handlers.py:193
msgid "Can't resume producing: not connected yet"
msgstr ""

#: src/vigilo/connector/handlers.py:289
#, python-format
msgid "Received message is not JSON-encoded: %r"
msgstr ""

#: src/vigilo/connector/handlers.py:726
#, python-format
msgid "Requeuing message (%(reason)s)."
msgstr ""

#: src/vigilo/connector/handlers.py:736
#, python-format
msgid "Error trying to save a message to the backup database: %s"
msgstr ""

#: src/vigilo/connector/handlers.py:798
#, python-format
msgid ""
"Can't understand the max_queue_size option, it should be an integer (or 0 for"
" no limit). Current value: %s"
msgstr ""

#: src/vigilo/connector/handlers.py:806
msgid "The max_queue_size option is only available on Python >= 2.6. Ignoring."
msgstr ""

#: src/vigilo/connector/handlers.py:814 src/vigilo/connector/socket.py:114
#, python-format
msgid "Directory not found: '%(dir)s'"
msgstr ""

#: src/vigilo/connector/handlers.py:819 src/vigilo/connector/socket.py:121
#, python-format
msgid "Wrong permissions on directory: '%(dir)s'"
msgstr ""

#: src/vigilo/connector/options.py:26
msgid "Load this settings.ini file"
msgstr ""

#: src/vigilo/connector/options.py:51
msgid "The configuration file does not exist"
msgstr ""

#: src/vigilo/connector/serialize.py:36
#, python-format
msgid "Got malformed message: %s"
msgstr ""

#: src/vigilo/connector/serialize.py:47 src/vigilo/connector/serialize.py:89
#, python-format
msgid "Unknown/malformed message type: '%s'"
msgstr ""

#: src/vigilo/connector/socket.py:40
#, python-format
msgid "Unparsable line: %s"
msgstr ""

#: src/vigilo/connector/ssl.py:102
#, python-format
msgid "Machine hostname (%(hostname)s) does not match certificate CommonName (%(cn)s)"
msgstr ""

#: src/vigilo/connector/status.py:61
msgid "OK: running"
msgstr ""

#: src/vigilo/connector/status.py:160
#, python-format
msgid "Stats for %(service)s: %(name)s = %(value)s"
msgstr ""

#: src/vigilo/connector/store.py:190 src/vigilo/connector/store.py:290
#, python-format
msgid "Could not fill the output buffer: %s"
msgstr ""

#: src/vigilo/connector/store.py:232
#, python-format
msgid "Could not VACUUM the database: %s"
msgstr ""

